core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamlet.scala

```diff
@@ -41,8 +42,9 @@ abstract class AkkaStreamlet extends Streamlet[AkkaStreamletContext] {
     (for {
       streamletDefinition ← StreamletDefinition.read(config)
     } yield {
-      val updatedConfig = streamletDefinition.copy(config = streamletDefinition.config.withFallback(config))
-      AkkaStreamletContextImpl(updatedConfig)
+      val updatedStreamletDefinition = streamletDefinition.copy(config = streamletDefinition.config.withFallback(config))
+      val system                     = ActorSystem("akka_streamlet", updatedStreamletDefinition.config)
```

Comment:
Why did we move system creation to the client/caller side?

Reply:
so you can use the test system in tests

Reply:
It should not be created in the context.